doc/sections/commands.md

```diff
@@ -207,6 +207,20 @@ arguments on the left side of the `≔` symbol (similarly to a value declaration
 in OCaml). Some arguments can be declared as implicit by enclosing them in
 curly brackets.
 
+<!---------------------------------------------------------------------------->
+### `inductive`
+The `inductive` command is used to define an inductive type, as in Coq.
+
+```
+inductive nat : TYPE ≔
+ | z    : nat
+ | succ : nat → nat
+
+inductive list : TYPE ≔
```

Comment:
One example is enough but give the code the command `inductive` is equivalent to:
```
constant symbol nat : TYPE
constant symbol z : nat
constant symbol succ : nat → nat
```

Reply:
done